sale_mrp_link/models/sale_order.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018 Alex Comba - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
```

Comment:
As the link in Production Orders is a normal Many2One field, could we simply add a One2Many field here for a proper link both ways?
Something like:
```suggestion
production_ids = fields.One2many('mrp.production','sale_order_id')
```


This would ease the functions below:
- production count calculation function (simple `len()` would be enough)
- action function, as we'll already have the production objects, hence no need to 'search' for them.